test/test.ts

```diff
@@ -0,0 +1,49 @@
+/// <reference path="../d/mocha.d.ts" />
+/// <reference path="../d/chai.d.ts" />
+/// <reference path="../d/jsdom.d.ts" />
+/// <reference path="../d/d3.d.ts" />
```

Comment:
@turboMaCk I've stumbled into this repository an I really like what you are doing here. 👏  👍  Just a question... Or a tip,.. I usually put references into `typings.d.ts`. In your case `test/typings.d.ts` usually TS compilers pick them up. So you don't have to "pollute" your pretty JS/TS code.